health_monitor/include/health_monitor.h

```diff
@@ -29,18 +29,41 @@ namespace health_monitor
     class HealthMonitor
     {
         public:
-
+            
+            /*!
+             * \brief Default constructor for HealthMonitor
+             */
+            HealthMonitor();
             /*!
              * \brief Begin normal execution of health monitor node. Will take over control flow of program and exit from here.
              * 
              * \return The exit status of this program
              */
             void run();
 
+            // spin callback function
+            bool spin_cb();
+
+            //Publisher Handling
+            std::vector<bool> getPubStatusCar();
+            std::vector<bool> getPubStatusTruck();
+
+            void spinObject(long time_now);
```

Comment:
Since this method does not seem to replace the spin_cb I think a more descriptive name would be more appropriate. What exactly is the method meant to do?